Migrators/Helpers/batchLoader.py

```diff
@@ -1,7 +1,25 @@
+from functools import partial
+from multiprocessing.dummy import Pool as ThreadPool
+from tqdm import tqdm
+
 from typedb.client import TransactionType
 
-def write_batch(session, batch):
+def write_batches(session, queries, batch_size, num_threads):
+    with ThreadPool(num_threads) as pool:
+        with tqdm(total=len(queries)) as pbar:
```

Comment:
Displays a progress bar that looks something like this:
`40%|██████████████▏                             | 453600/1134942 [04:05<03:23, 3341.85it/s]`